examples/cifar10/cifar10-real-world/README.md

```diff
@@ -60,10 +60,14 @@ cd ..
 ```
 For more information about secure provisioning see the [documentation](https://nvflare.readthedocs.io/en/latest/programming_guide/provisioning_system.html).
 
-### 3.3 Multi-tasking resource management
-In this example, we assume two local GPUs with at least 8 GB of memory are available. 
-
-Hence, we need to change the clients' local `GPUResourceManager` configurations to show two GPUs at each client, 
+### 3.2 Multi-tasking resource management
+In this example, we assume `N_GPU` local GPUs with at least 8 GB of memory 
```

Comment:
maybe further clarify the memory requirement: 
each client need 1 GB of memory, therefore the minimum combined memory is 8 GB for 8 clients.

Reply:
It's explained a bit below.

Reply:
@ZiyueXu77, I rephrased it a bit to be clearer.